client/components/web-preview/style.scss

```diff
@@ -141,12 +141,6 @@ a.web-preview__external.button {
 	display: flex;
 	flex-wrap: wrap;
```

Comment:
I saw this and though the same thing - the toolbar is a fixed height right? Any wrapping would push elements over the bounds of the bar. I figured not to touch it in case there was some good reason for it that I hadn't thought of... I can't think of any though.